atlasdb-api/src/main/java/com/palantir/atlasdb/config/LeaderConfig.java

```diff
@@ -79,6 +79,8 @@ public long leaderPingResponseWaitMs() {
 
     @Value.Check
     protected final void check() {
+        Preconditions.checkState(quorumSize() == 1 && leaders().size() > 2,
```

Comment:
The only time quorom should be 1 is if there is exactly 1 leader, so should be `leaders().size() > 1`